legacy-docs/docs/d/fabric_node_member.html.markdown

```diff
@@ -21,17 +30,17 @@ data "aci_fabric_node_member" "example" {
 
 ## Argument Reference
 
-- `serial` - (Required) serial of Object fabric_node_member.
+- `serial` - (Required) Serial Number for the new Fabric Node Member. Type: String.
 
 ## Attribute Reference
 
-- `id` - Attribute id set to the Dn of the Fabric Node Member.
-- `annotation` - (Optional) annotation for object fabric_node_member.
-- `ext_pool_id` - (Optional) ext_pool_id for object fabric_node_member.
-- `fabric_id` - (Optional) place holder for a value
-- `name_alias` - (Optional) name_alias for object fabric_node_member.
-- `node_id` - (Optional) node id
-- `node_type` - (Optional) node_type for object fabric_node_member.
-- `pod_id` - (Optional) pod id
-- `role` - (Optional) system role type
-- `serial` - (Optional) serial number
+- `id` - (Read-Only) Attribute id set to the Dn of the Fabric Node Member. Type: String.
```

Comment:
I see you use capitalised ID everywhere except here and for pod
```suggestion
- `id` - (Read-Only) Attribute ID set to the DN of the Fabric Node Member. Type: String.
```

Reply:
changed